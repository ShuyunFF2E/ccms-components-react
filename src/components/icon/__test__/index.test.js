import React from 'react';
import { render } from 'enzyme';
import Icon from '../index';

describe('Icon', () => {

  it('renders correctly', () => {
    const wrapper = render(<Icon type="config"/>);
    expect(wrapper).toMatchSnapshot();
  })

  it('should render correctly has style', () => {
    const style = {
      color: '#0083BA',
      fontSize: 12
    }
    const wrapper = render(<Icon type="edit" style={style}/>)
    expect(wrapper).toMatchSnapshot();
  })
});
