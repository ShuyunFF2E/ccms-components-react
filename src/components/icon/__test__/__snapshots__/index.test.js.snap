// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon renders correctly 1`] = `
<i
  class="cloud-icon icon-config"
/>
`;

exports[`Icon should render correctly has style 1`] = `
<i
  class="cloud-icon icon-edit"
  style="color:#0083BA;font-size:12px"
/>
`;
